### PR TITLE
Move OpenAxon repo to Axon, CI

### PR DESCRIPTION
**Pull request by GrantPerkins**
This allows for sections of the OpenImages dataset to be downloaded and used in Axon.

### CODE REVIEW

**Comment by GrantPerkins** [comment]
This PR should build the OpenImages docker image in CI.

File: containers/utils/openimages/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get install python python3-pip python-dev python3-opencv -yq && \
+    pip3 install openimages pandas
+
+COPY . /down/
+
+WORKDIR /down/
+
+ENTRYPOINT ["python3", "download.py"]
```
Comment (AustinShalit) [inline containers/utils/openimages/Dockerfile]: eol @ eof

File: containers/utils/openimages/download.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                        # Add file to zip
+                        zipFile.write("tar/" + directory + '/' + filename, file)
+        print("done.")
+
+
+if __name__ == "__main__":
+    data_json = sys.argv[1]
+    downloader = OpenImagesDownloader(data_json)
+    # downloader.download()
+    # downloader.create_csv()
```
Comment (AustinShalit) [inline containers/utils/openimages/download.py]: Do we need these lines?

File: containers/utils/openimages/download.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+        with open("tar/{}/_annotations.csv".format(name), 'w+') as csv:
+            csv.write("filename,width,height,class,xmin,ymin,xmax,ymax\n")
+            r = range(len(self.csv))[:int(len(self.csv) * .7)] if name == "train" else range(len(self.csv))[
+                                                                                       int(len(self.csv) * .7):]
+            for i in r:
+                row = self.csv[i]
+                csv.write("{},{},{},{},{},{},{},{}\n".format(row["filename"], row["width"], row["height"], row["class"],
+                                                             row["xmin"], row["ymin"], row["xmax"], row["ymax"]))
+                copyfile("tar/" + row["filename"], "tar/" + name + '/' + row["filename"])
+                # self.label_frame("tar/" + row["filename"], row["class"], row["xmin"], row["xmax"], row["ymin"], row["ymax"])
```
Comment (AustinShalit) [inline containers/utils/openimages/download.py]: Do we need this line? If it is for debug, can we stick it in an if statement?

File: containers/utils/openimages/run.bat [hunk 4]
```diff
@@ -0,0 +1 @@
+docker run --mount type=bind,source="%cd%",target=/down -i -t down data.json
```
Comment (AustinShalit) [inline containers/utils/openimages/run.bat]: Do we need a cross platform version of this?

**Review by AustinShalit — CHANGES_REQUESTED** [review]
Seems like all of these files should be up 1 level at `containers/openimages`.

Please post a photo showing that this works as expected.

File: containers/utils/openimages/run.bat (hunk 4 above)
Comment (GrantPerkins) [reply containers/utils/openimages/run.bat]: I'm implementing this to be run within the GUI. just wanted to get the CI set up.

File: containers/utils/openimages/download.py (hunk 2 above)
Comment (GrantPerkins) [reply containers/utils/openimages/download.py]: yes, haha.

File: containers/utils/openimages/download.py (hunk 3 above)
Comment (GrantPerkins) [reply containers/utils/openimages/download.py]: will remove.

**Comment by GrantPerkins** [comment]
> 
> 
> Seems like all of these files should be up 1 level at `containers/openimages`.
> 
> Please post a photo showing that this works as expected.

What kind of photo do you want? I have Alex using Axon to recognize coffee cups using this, if that's good enough.

**Comment by AustinShalit** [comment]
> What kind of photo do you want? I have Alex using Axon to recognize coffee cups using this, if that's good enough.

That good. Just want to make sure someone tested it because I can't right now.

**Comment by GrantPerkins** [comment]
Pic coming, @technoAl

**Comment by GrantPerkins** [comment]
image

Need to fix a bug with inference where spaces in class name causes letters to be cut.

**Review by AustinShalit — APPROVED** [review]

**Pull request merged**